libshpool/src/lib.rs

```diff
@@ -79,7 +80,7 @@ running in daemon mode, the logs will go to stderr by default."
         action,
         long_help = "The path for the unix socket to listen on
 
-This defaults to $XDG_RUNTIME_DIR/shpool/shpool.socket or ~/.shpool/shpool.socket
+This defaults to $XDG_RUNTIME_DIR/shpool/shpool.socket or ~/.local/run/shpool.socket
```

Comment:
The comment says `~/.local/run/shpool.socket`, but the path is `~/.local/run/shpool/shpool.socket`. (I think it is good to use a subdirectory in both `$XDG_RUNTIME_DIR` and  `~/.local/run/`. I suggest updating the comment, not the code.)

Reply:
Oh yeah, good catch, thanks